package.json

```diff
@@ -6,7 +6,9 @@
   "author": "muhwezi deo <aggrey256@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "start": "tsc && node dist/index.js"
+    "start": "tsc && node dist/index.js",
+    "heroku-prebuild": "npx rimraf package-lock.json npm-shrinkwrap.json node_modules && npm install yarn && yarn install",
```

Comment:
what is this script

Reply:
Man, ain't asked your review yet.
Besides, as the name suggests, prebuild installs yarn then yarn installs the packages. yarn is technically lighter than npm hence the script.

Reply:
Thanks